docs/stdlib/datetime.rst

```diff
@@ -905,6 +903,93 @@ EdgeDB stores and outputs timezone-aware values in UTC.
         ...   <duration>'30 hours', 'totalseconds');
         {108000}
 
+    .. rubric:: Gotchas
+
+    This function will provide you with a calculated total for the unit passed
+    as ``el``, but only within the given "size class" of the unit. These size
+    classes exist because they are logical breakpoints that we can't reliably
+    convert values across. A month might be 30 days long, or it might be 28 or
+    29 or 31. A day is generally 24 hours, but with daylight savings, it might
+    be longer or shorter.
+
+    As a result, it's impossible to convert across these lines in a way that
+    works in every situation. For some use cases, assuming a 30 day month works
+    fine. For others, it might not. The size classes are as follows:
+
+    - ``'month'`` and larger
+    - ``'day'``
+    - ``'hour'`` and smaller
+
+    For example, if you specify ``'day'`` as your ``el`` argument, the function
+    will return only the number of days expressed as ``N days`` in your
+    duration. It will not add another day to the returned count for every 24
+    hours (defined as ``24 hours``) in the duration, nor will it consider the
+    months' constituent day counts in the returned value.
+
+    Specifying ``'decade'`` for ``el`` will total up all decades represented in
+    units ``'month'`` and larger, but it will not add a decade's worth of days
+    to the returned value as an additional decade. On the other end of the unit
+    spectrum, passing ``'hour'`` for ``el`` will not add a day's worth of hours
+    for each day in your duration.
```

Comment:
The part about hours not rolling over into days seems to be a repetition of the point made in the previous paragraph.

Reply:
You're right on this. I removed that sentence and joined the one about decades with the previous paragraph.